File: blotztask-api/Modules/Tasks/DTOs/AddTaskItemDTO.cs
namespace BlotzTask.Modules.Tasks.DTOs;

public class AddTaskItemDto
{
    public string Title { get; set; }
    public string Description { get; set; }
    public DateTimeOffset? EndTime { get; set; }
    public DateTimeOffset? StartTime { get; set; }
    public int LabelId { get; set; }
    public bool HasTime { get; set; }
}

File: blotztask-mobile/src/feature/task/models/add-task-item-dto.ts
import { TaskTimeType } from "../../task-add-edit/util/time-type-mapper";

export interface AddTaskItemDTO {
  title: string;
  description?: string;
  startTime?: Date;
  endTime?: Date;
  timeType: number;
  labelId: number;
  timeType?: TaskTimeType;
}
